src/main/java/neuefische/capstone/backend/rest/ressource/CpGroupRes.java

```diff
@@ -1,15 +1,14 @@
-package com.example.devcapstonebackend.ressource;
+package neuefische.capstone.backend.rest.ressource;
 
-import java.util.List;
-
-import com.example.devcapstonebackend.model.CodepointGroup;
+import neuefische.capstone.backend.rest.model.CodepointGroup;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
+import java.util.List;
+
 @RepositoryRestResource(collectionResourceRel = "codepoint-groups", path = "codepoint-groups")
 public interface CpGroupRes extends PagingAndSortingRepository<CodepointGroup, Long> {
-
     List<CodepointGroup> findByName(@Param("name") String name);
```

Comment:
what is this @Param doing?

Reply:
telling Spring Data Rest to bind the method variable name to querry param name in the created controller